django_project/ford3/views/qualification_wizard.py

```diff
@@ -80,28 +65,61 @@ def validate_form_data(self, form_data):
                 )
             except AttributeError:
                 continue
+        return qualification_fields
+
+    def process_data(self, form_data):
```

Comment:
This function is too long, does too many things and is really complex to read.
We could have a tiny function for each qualification field.